### PR TITLE
docs: updates with latest logs change

### PR DESCRIPTION
**Pull request by aarnphm**
update tutorials with latest log changes

ideally we should use some sort of repl.it for all of this code block that can
be ran on the docs. Logging can then be updated correctly.

I updated this for now so that it fits with rc3 logs.

### CODE REVIEW

File: docs/source/concepts/model.rst [hunk 1]
```diff
@@ -12,10 +12,10 @@ pipeline, invoking a :code:`save_model` call, as demonstrated in the
 
 .. code:: python
 
-    bentoml.sklearn.save_model("iris_clf", clf)
+   bentoml.sklearn.save_model("iris_clf", clf)
 
-    # INFO  [cli] Using default model signature `{"predict": {"batchable": False}}` for sklearn model
-    # INFO  [cli] Successfully saved Model(tag="iris_clf:2uo5fkgxj27exuqj", path="~/bentoml/models/iris_clf/2uo5fkgxj27exuqj/")
+   # INFO  [cli] Using default model signature `{"predict": {"batchable": False}}` for sklearn model
```
Comment (sauyon) [inline docs/source/concepts/model.rst:17]: This doesn't seem right? Why are these spaces being removed?

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -141,32 +140,34 @@ a file :code:`service.py` with:
...
-    INFO [cli] Starting development BentoServer from "service:svc" running on http://127.0.0.1:3000 (Press CTRL+C to quit)
-    INFO [dev_api_server] Service imported from source: bentoml.Service(name="iris_classifier", import_str="service:svc", working_dir="/home/user/gallery/quickstart")
-    INFO [dev_api_server] Will watch for changes in these directories: ['/home/user/gallery/quickstart']
-    INFO [dev_api_server] Started server process [25915]
-    INFO [dev_api_server] Waiting for application startup.
-    INFO [dev_api_server] Application startup complete.                                                                                                                          on.py:59
+   2022-07-01T14:19:12-0700 [INFO] [] Starting development BentoServer from "service:svc" running on http://127.0.0.1:3000 (Press CTRL+C to quit)
+   2022-07-01 14:19:12 circus[10959] [INFO] Loading the plugin...
+   2022-07-01 14:19:12 circus[10959] [INFO] Endpoint: 'tcp://127.0.0.1:52870'
+   2022-07-01 14:19:12 circus[10959] [INFO] Pub/sub: 'tcp://127.0.0.1:52871'
```
Comment (sauyon) [inline docs/source/tutorial.rst:166]: I think you're running in debug mode.

File: docs/source/concepts/model.rst (hunk 1 above)
Comment (aarnphm) [reply docs/source/concepts/model.rst:17]: oo let me disable formatter.

File: docs/source/concepts/model.rst [hunk 3]
```diff
@@ -12,12 +12,13 @@ pipeline, invoking a :code:`save_model` call, as demonstrated in the
 
 .. code:: python
 
-    bentoml.sklearn.save_model("iris_clf", clf)
+    saved_model = bentoml.sklearn.save_model("iris_clf", clf)
+    print(f"Model saved: {saved_model}")
 
-    # INFO  [cli] Using default model signature `{"predict": {"batchable": False}}` for sklearn model
-    # INFO  [cli] Successfully saved Model(tag="iris_clf:2uo5fkgxj27exuqj", path="~/bentoml/models/iris_clf/2uo5fkgxj27exuqj/")
+    # Model saved: Model(tag="iris_clf:d2f2vuxzrsetzgxi")
```
Comment (parano) [inline docs/source/concepts/model.rst:18]:
```suggestion
    # Model saved: Model(tag="iris_clf:2uo5fkgxj27exuqj")
```

Comment (parano) [reply docs/source/concepts/model.rst:18]: let's just keep it same as the readme and Quickstart

File: docs/source/concepts/model.rst [hunk 4]
```diff
@@ -171,13 +174,14 @@ stages. For example:
 
     > bentoml models export iris_clf:latest .
 
-    INFO [cli] Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ./iris_clf-2uo5fkgxj27exuqj.bentomodel
+    Model(tag="iris_clf:d2f2vuxzrsetzgxi") exported to ~/workspace/iris_clf-d2f2vuxzrsetzgxi.bentomodel
```
Comment (parano) [inline docs/source/concepts/model.rst:177]:
```suggestion
    Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ~/workspace/iris_clf-2uo5fkgxj27exuqj.bentomodel
```

File: docs/source/concepts/model.rst [hunk 5]
```diff
@@ -171,13 +174,14 @@ stages. For example:
...
 
 .. code:: bash
 
     > bentoml models import ./iris_clf-2uo5fkgxj27exuqj.bentomodel
 
-    INFO [cli] Model(tag="iris_clf:2uo5fkgxj27exuqj") imported
+    Successfully saved Model(tag="iris_clf:d2f2vuxzrsetzgxi")
+    Model(tag="iris_clf:d2f2vuxzrsetzgxi") imported
```
Comment (parano) [inline docs/source/concepts/model.rst:184]:
```suggestion
    Successfully saved Model(tag="iris_clf:2uo5fkgxj27exuqj")
    Model(tag="iris_clf:2uo5fkgxj27exuqj") imported
```

File: docs/source/tutorial.rst [hunk 6]
```diff
@@ -101,8 +102,7 @@ managing all your trained models locally as well as accessing them for serving.
    # Save model to the BentoML local model store
    bentoml.sklearn.save_model("iris_clf", clf)
```
Comment (parano) [inline docs/source/tutorial.rst:103]:
```suggestion
   bentoml.sklearn.save_model("iris_clf", clf)
   print(f"Model saved: {saved_model}")
```

File: docs/source/tutorial.rst [hunk 7]
```diff
@@ -100,9 +101,10 @@ managing all your trained models locally as well as accessing them for serving.
 
    # Save model to the BentoML local model store
    bentoml.sklearn.save_model("iris_clf", clf)
+   print(f"Model saved: {saved_model}")
+   print(f"Model saved: {saved_model}")
```
Comment (parano) [inline docs/source/tutorial.rst:105]:
```suggestion
```

File: docs/source/tutorial.rst [hunk 8]
```diff
@@ -372,11 +373,13 @@ via the :code:`bentoml containerize` CLI command:
 
 .. code:: bash
 
-   > bentoml containerize iris_classifier:latest
+    > bentoml containerize iris_classifier:latest
```
Comment (parano) [inline docs/source/tutorial.rst:376]:
```suggestion
   > bentoml containerize iris_classifier:latest
```

File: docs/source/tutorial.rst [hunk 9]
```diff
@@ -372,11 +373,13 @@ via the :code:`bentoml containerize` CLI command:
...
 
-   INFO  [cli] Successfully built docker image "iris_classifier:dpijemevl6nlhlg6"
+    Building docker image for Bento(tag="iris_classifier:6otbsmxzq6lwbgxi")...
+    Successfully built docker image "iris_classifier:6otbsmxzq6lwbgxi"
```
Comment (parano) [inline docs/source/tutorial.rst:379]:
```suggestion
   Building docker image for Bento(tag="iris_classifier:6otbsmxzq6lwbgxi")...
   Successfully built docker image "iris_classifier:6otbsmxzq6lwbgxi"
```

File: docs/source/tutorial.rst [hunk 10]
```diff
@@ -318,18 +325,19 @@ Next, run the :code:`bentoml build` CLI command from the same directory:
...
-    INFO [cli]
-         ██████╗░███████╗███╗░░██╗████████╗░█████╗░███╗░░░███╗██╗░░░░░
-         ██╔══██╗██╔════╝████╗░██║╚══██╔══╝██╔══██╗████╗░████║██║░░░░░
-         ██████╦╝█████╗░░██╔██╗██║░░░██║░░░██║░░██║██╔████╔██║██║░░░░░
-         ██╔══██╗██╔══╝░░██║╚████║░░░██║░░░██║░░██║██║╚██╔╝██║██║░░░░░
-         ██████╦╝███████╗██║░╚███║░░░██║░░░╚█████╔╝██║░╚═╝░██║███████╗
-         ╚═════╝░╚══════╝╚═╝░░╚══╝░░░╚═╝░░░░╚════╝░╚═╝░░░░░╚═╝╚══════╝
-                                                                                                                                                                                                                                                                                                                        
-    INFO [cli] Successfully built Bento(tag="iris_classifier:dpijemevl6nlhlg6") at "~/bentoml/bentos/iris_classifier/dpijemevl6nlhlg6/"
+    Building BentoML service "iris_classifier:6otbsmxzq6lwbgxi" from build context "/Users/aarnphm/workspace/bentoml/bentoml_playground"
```
Comment (parano) [inline docs/source/tutorial.rst:328]:
```suggestion
    Building BentoML service "iris_classifier:6otbsmxzq6lwbgxi" from build context "/home/user/gallery/quickstart"
```

File: docs/source/tutorial.rst [hunk 11]
```diff
@@ -161,12 +163,14 @@ Run it live:
...
-    INFO [dev_api_server] Started server process [25915]
-    INFO [dev_api_server] Waiting for application startup.
-    INFO [dev_api_server] Application startup complete.                                                                                                                          on.py:59
+    2022-07-01T14:19:12-0700 [INFO] [] Starting development BentoServer from "service:svc" running on http://127.0.0.1:3000 (Press CTRL+C to quit)
+    2022-07-01 14:19:12 circus[10959] [INFO] Loading the plugin...
+    2022-07-01 14:19:12 circus[10959] [INFO] Endpoint: 'tcp://127.0.0.1:52870'
+    2022-07-01 14:19:12 circus[10959] [INFO] Pub/sub: 'tcp://127.0.0.1:52871'
+    2022-07-01 14:19:12 asyncio[10959] [DEBUG] Using selector: KqueueSelector
+    2022-07-01 14:19:12 asyncio[10959] [DEBUG] Using selector: KqueueSelector
+    2022-07-01 14:19:12 bentoml._internal.utils.circus[10959] [INFO] Watching directories: /Users/aarnphm/workspace/bentoml/bentoml_playground
```
Comment (parano) [inline docs/source/tutorial.rst:172]:
```suggestion
    2022-07-01 14:19:12 bentoml._internal.utils.circus[10959] [INFO] Watching directories: /home/user/gallery/quickstart
```

Comment (parano) [inline docs/source/tutorial.rst:172]:
```suggestion
    2022-07-01 14:19:12 bentoml._internal.utils.circus[10959] [INFO] Watching directories: /home/user/gallery/quickstart
```

File: docs/source/concepts/model.rst [hunk 12]
```diff
@@ -171,13 +174,14 @@ stages. For example:
 
     > bentoml models export iris_clf:latest .
 
-    INFO [cli] Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ./iris_clf-2uo5fkgxj27exuqj.bentomodel
+    Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ~/workspace/iris_clf-2uo5fkgxj27exuqj.bentomodel
```
Comment (parano) [inline docs/source/concepts/model.rst:177]:
```suggestion
    Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ./iris_clf-2uo5fkgxj27exuqj.bentomodel
```

Comment (parano) [reply docs/source/concepts/model.rst:177]: this depends on users' current directory

File: docs/source/tutorial.rst [hunk 13]
```diff
@@ -318,18 +324,19 @@ Next, run the :code:`bentoml build` CLI command from the same directory:
...
-    INFO [cli]
-         ██████╗░███████╗███╗░░██╗████████╗░█████╗░███╗░░░███╗██╗░░░░░
-         ██╔══██╗██╔════╝████╗░██║╚══██╔══╝██╔══██╗████╗░████║██║░░░░░
-         ██████╦╝█████╗░░██╔██╗██║░░░██║░░░██║░░██║██╔████╔██║██║░░░░░
-         ██╔══██╗██╔══╝░░██║╚████║░░░██║░░░██║░░██║██║╚██╔╝██║██║░░░░░
-         ██████╦╝███████╗██║░╚███║░░░██║░░░╚█████╔╝██║░╚═╝░██║███████╗
-         ╚═════╝░╚══════╝╚═╝░░╚══╝░░░╚═╝░░░░╚════╝░╚═╝░░░░░╚═╝╚══════╝
-                                                                                                                                                                                                                                                                                                                        
-    INFO [cli] Successfully built Bento(tag="iris_classifier:dpijemevl6nlhlg6") at "~/bentoml/bentos/iris_classifier/dpijemevl6nlhlg6/"
+    Building BentoML service "iris_classifier:dpijemevl6nlhlg6" from build context "/home/user/gallery/quickstart"
```
Comment (parano) [inline docs/source/tutorial.rst:327]:
```suggestion
    Building BentoML service "iris_classifier:6otbsmxzq6lwbgxi" from build context "/home/user/gallery/quickstart"
```

File: docs/source/tutorial.rst [hunk 14]
```diff
@@ -318,18 +324,19 @@ Next, run the :code:`bentoml build` CLI command from the same directory:
...
+    Packing model "iris_clf:zy3dfgxzqkjrlgxi"
+    BentoML is installed in `editable` mode; building BentoML distribution with the local BentoML code base. The built wheel file will be included in the target bento.
```
Comment (parano) [inline docs/source/tutorial.rst:329]:
```suggestion
```

File: docs/source/concepts/model.rst [hunk 15]
```diff
@@ -171,13 +174,14 @@ stages. For example:
...
 
-    INFO [cli] Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ./iris_clf-2uo5fkgxj27exuqj.bentomodel
+    Model(tag="iris_clf:2uo5fkgxj27exuqj") exported to ./iris_clf-2uo5fkgxj27exuqj.bentomodel
 
 .. code:: bash
 
     > bentoml models import ./iris_clf-2uo5fkgxj27exuqj.bentomodel
 
-    INFO [cli] Model(tag="iris_clf:2uo5fkgxj27exuqj") imported
+    Successfully saved Model(tag="iris_clf:2uo5fkgxj27exuqj")
```
Comment (parano) [inline docs/source/concepts/model.rst:183]:
```suggestion
```

Comment (parano) [reply docs/source/concepts/model.rst:183]: This is an unexpected log, we should remove it.

**Pull request merged**